View/src/observer.h

```diff
@@ -0,0 +1,37 @@
+/*
+ * observer.h
+ *
+ *  Created on: Feb 14, 2015
+ *      Author: edem
+ */
+
+#ifndef OBSERVER_H_
+#define OBSERVER_H_
+
+/*
+ * This abstract class is informed by the
+ * observable class when observable changes state.
+ * It then retrieves the data once it is available
+ * to read.
+ */
+
+class Observer
+{
+public:
+		Observer();
+
+		bool freeToRead();
+		void readState();
+
```

Comment:
I think this class needs some sort of virtual update function, so that the observable can notify you of a change in State